src/Playlist.java

```diff
@@ -0,0 +1,86 @@
+package components.Playlist;
+
+/**
+ * {@code SequenceKernel} enhanced with secondary methods.
+ *
+ * @param <String>
+ *            type of {@code Sequence} entries
+ */
+public interface Playlist<String> extends PlaylistKernel<String> {
```

Comment:
You don't need the generic type. You would only use that if you wanted anyone to store anything in your playlist. 
```suggestion
public interface Playlist extends PlaylistKernel {
```